config/samples/simplekafkacluster_affinity.yaml

```diff
@@ -0,0 +1,263 @@
+apiVersion: kafka.banzaicloud.io/v1beta1
+kind: KafkaCluster
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: kafka
+spec:
+  headlessServiceEnabled: true
+  zkAddresses:
+    - "zookeeper-client.zookeeper:2181"
+  propagateLabels: false
+  oneBrokerPerNode: false
```

Comment:
Add some comment explaining that this field is ignored if `affinity` is set either in broker config group or broker config